clients/admin-ui/src/features/system/SystemInformationForm.tsx

```diff
@@ -612,6 +612,14 @@ const SystemInformationForm = ({
                   id="legitimate_interest_disclosure_url"
                   disabled={lockedForGVL}
                 />
+                <DictSuggestionTextInput
+                  label="Vendor deleted date"
+                  name="vendor_deleted_date"
+                  id="vendor_deleted_date"
+                  // disable this field for editing:
+                  // deleted date is populated by the GVL and should not be editable by users
+                  disabled
```

Comment:
i decided to lock down this field on _all_ systems, since allowing it to be editable for non-GVL systems would lead to a UI footgun, as it's not a proper "datetime" field, so when a user tries to edit it, it inevitably fails to save/validate.

the proper fix would probably be to have a `DictSuggestionDateInput` field, but that's above my know-how, and probably beyond the immediate scope here.

what's important is that the field populates well with the GVL vendors that have a `vendor_deleted_date`, both in the UI form as well as what gets sent to the BE and persisted to the DB 👍